README.md

```diff
@@ -97,10 +97,6 @@ There is a new configuration class which is not backwards compatible with the co
 
 In 4.x we use DDLog (Lumberjack) for our logging, and therefore, logging configuration has changed. Please see "Logging" below for more information on how to use the new logger.
```

Comment:
Shouldn't we also remove the note saying we use `DDLog`?

Reply:
Strange, it should be removed in previous patches which already has been merged to develop, but it still there.